EpsilonLights/Core/Src/Lights.c

```diff
@@ -40,14 +38,14 @@ void updateLightsTask(void const* arg)
         {
             HAL_GPIO_WritePin(HEAD_GPIO_Port, HEAD_Pin, LIGHT_OFF);
         }
-        else if ((headlightsLow && headlightsHigh))
+        else if ((headlights))
         {
             // Error state, turn only the low headlights on.
             HAL_GPIO_WritePin(HEAD_GPIO_Port, HEAD_Pin, LIGHT_ON);
         }
         else
         {
-            HAL_GPIO_WritePin(HEAD_GPIO_Port, HEAD_Pin, headlightsLow);
+            HAL_GPIO_WritePin(HEAD_GPIO_Port, HEAD_Pin, headlights);
```

Comment:
I think you can just turn the headlights off in this condition

Reply:
I think you can do
```
if(headlightsOff)
{...}
else
{
HAL_GPIO_WritePin(HEAD_GPIO_Port, HEAD_Pin, headlights);
}
```